server/models/post.ts

```diff
@@ -33,6 +33,7 @@ export class Post
   UserId!: string; /* Foreign Key from UserId */
 
   static associate(model: any) {
+    Post.belongsTo(model.User);
```

Comment:
User has 'hasMany' association to Post. Is this fine? I think we should check what the db looks like before merging

Reply:
This is needed.